src/toggle-switch/toggle-switch.scss

```diff
@@ -94,47 +110,58 @@
           border-color: t(iui-color-background-primary);
         }
 
-        > .iui-icon {
-          @include themed {
-            opacity: t(iui-opacity-1);
-          }
-        }
-
-        > .iui-handle {
+        &::after {
+          left: $iui-m + $iui-xs + 1px;
           @include themed {
             background-color: t(iui-color-foreground-accessory);
             opacity: t(iui-opacity-2);
           }
-          right: $iui-xxs;
         }
-      }
 
-      &:hover:not(:disabled) {
-        ~ .iui-toggle {
+        > .iui-icon {
           @include themed {
-            border-color: t(iui-color-background-primary);
+            opacity: t(iui-opacity-1);
           }
+        }
+      }
 
-          > .iui-handle {
-            @include themed {
-              opacity: t(iui-opacity-1);
-            }
-          }
+      &:hover:enabled ~ .iui-toggle {
+        @include themed {
+          border-color: t(iui-color-background-primary);
         }
       }
     }
 
+    // Focused or hovered toggle switch
+    &:hover:enabled ~ .iui-toggle::after,
```

Comment:
Focus changed?
current
![image](https://user-images.githubusercontent.com/81580355/151776395-4e81383c-0faf-4cfe-af61-1785476324d2.png)


deployed
![image](https://user-images.githubusercontent.com/81580355/151776415-5c599229-aa13-4c77-981e-7767412ddee9.png)

Also, if you press on toggle (from active) and back, you can see white color now stays when you move mouse out.